index.ios.js

```diff
@@ -29,27 +18,18 @@ const store = createStore(
   applyMiddleware(thunk, promise, logger)
 );
 
-class findAR extends Component {
-
-  constructor() {
-    super();
-  }
+const scenes = routerActions.create(
+  <Scene key="root" hideNavBar>
+    <Scene key="ar" component={ARView} />
+    <Scene initial key="map" component={MapView} />
+    <Scene key="list" component={PinListView} />
```

Comment:
I think we can implement those routes as we add the appropriate features. Like, once we set up a sign in component, we can change 'initial' to whatever that route is, and then have it transition to map when you sign in without giving a 'back' button.

I would be hesitant to add all potential routes before there is anything on the other side of them in this PR, but we should definitely plan for them.